client/me/purchases/manage-purchase/index.jsx

```diff
@@ -500,6 +541,203 @@ class ManagePurchase extends Component {
 		return null;
 	}
 
+	renderRemoveSubscriptionWarningDialog( site, purchase ) {
+		if ( this.state.showRemoveSubscriptionWarningDialog ) {
+			const { hasCustomPrimaryDomain, primaryDomain } = this.props;
+			const primaryDomainName = hasCustomDomain && primaryDomain ? primaryDomain.name : '';
+			const isPlanRefundable = isRefundable( purchase );
+			const actionFunction = isPlanRefundable
+				? this.goToCancelLink
+				: this.showPreCancellationModalDialog;
+
+			let customDomain = false;
+			let wordpressComURL = '';
+
+			if ( hasCustomPrimaryDomain && primaryDomainName ) {
+				customDomain = true;
+			}
+
+			if ( typeof site.wpcom_url !== 'undefined' ) {
+				wordpressComURL =
+					site.wpcom_url.length < 35
+						? site.wpcom_url
+						: site.wpcom_url.substr( 0, 10 ) + '…' + site.wpcom_url.slice( -20 );
+			}
```

Comment:
See the comment regarding the same preprocessing below :)

Reply:
this is fixed, see comment on the other occurrence